src/Bridges/Constraint/bridges/norm_special_case.jl

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    NormSpecialCaseBridge{T,F} <: Bridges.Constraint.AbstractBridge
+
+`NormSpecialCaseBridge` implements the following reformulations:
+
+  * ``(t, x) in NormOneCone(d)`` into ``(t, x) in NormCone(1, d)``
+  * ``(t, x) in SecondOrderCone(d)`` into ``(t, x) in NormCone(2, d)``
+  * ``(t, x) in NormInfinityCone(d)`` into ``(t, x) in NormCone(Inf, d)``
+
+## Source node
+
+`NormSpecialCaseBridge` supports:
+
+  * `F` in [`MOI.NormOneCone`](@ref)
+  * `F` in [`MOI.NormInfinityCone`](@ref)
+  * `F` in [`MOI.SecondOrderCone`](@ref)
+
+## Target nodes
+
+`NormSpecialCaseBridge` creates:
+
+  * `F` in [`MOI.NormCone`](@ref)
+"""
+struct NormSpecialCaseBridge{T,F} <: AbstractBridge
```

Comment:
It would be easier to implement it as a `SetMapBridge`

Reply:
Refactored in to SetMapBridge